php/EE/Migration/Executor.php

```diff
@@ -2,28 +2,26 @@
 
 namespace EE\Migration;
 
-use \EE;
-use \EE\Model\Migration;
-use \EE\Utils;
-use Symfony\Component\Finder\Finder;
+use EE;
+use EE\Model\Migration;
+use EE\Utils;
 
 class Executor {
 
-    const MIGRATION_PATH = EE_ROOT . '/migrations';
+    const MIGRATION_PATH = EE_ROOT . '/migrations/';
 
     /**
      * Executes all pending migrations
      */
     public static function execute_migrations() {
 
         Utils\delem_log( "ee migration start" );
-        EE::log( "Migrating EasyEngine data to new version" );
+        EE::log( "Executing migrations" );
 
         $migrations = self::get_migrations_to_execute();
 
         if( empty( $migrations ) ) {
             EE::success( "Noting to migrate" );
```

Comment:
Should not be printed in the first execution. 
Also, take care of the same in this success message: https://github.com/EasyEngine/easyengine/blob/2ce1a036e76540623ac39a4db091f4a422a27f10/php/EE/Migration/Executor.php#L36